test/core/utils/preload-cssom.js

```diff
@@ -0,0 +1,120 @@
+describe('axe.utils.preloadCssom', function () {
```

Comment:
This is missing a bunch of tests:
- Does it actually return the right style sheets
- Does it make requests and than return stylesheets that were fetched
- Does it properly fetch all shadow DOM stylesheets as well
- Does it ignore disabled style sheets